ui/src/Elements/Message/README.md

```diff
@@ -55,5 +55,3 @@ The full set of exposed components and constants are as follows:
 
 - `ProducerMessage` - Message component with producer styling/structure
 - `ConsumerMessage` - Message component with consumer styling/structure
-- `PRODUCER` - producer usage constant
```

Comment:
Makes sense since we only export the HoC versions where this option is taken away